app/views/assignments/_current.haml

```diff
@@ -1,11 +1,11 @@
-#current_assignment
+#current-assignment
   - if @current_assignment.present?
-    %h3.slightly-green Currently on call: #{@current_assignment.user.last_name}
+    %h3.vague-success Currently on call: #{@current_assignment.user.last_name}
```

Comment:
Maybe the person on call makes a big mistake? Then they're only vaguely successful at their assignment, right?